Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,20 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const calc = tasks
+    .map(task => ({ name: task.name, duration: task.duration / 60 })) // transforms minutes to hours
+    .filter(time => time.duration >= 2) // filter array task duration equal and greater than 2
+    .reduce((acc, curr) => acc + curr.duration * hourlyRate, 0); // add duration * hourly rate values to the acc
+
+  return calc;
```

Comment:
Quite nice - clean and concise. I like it.

However, I am not too happy about the variable names `calc` and `curr`. I think `totalEarnings` and `task` respectively might better describe your intentions. 😉